numpy/lib/function_base.py

```diff
@@ -4956,16 +4956,25 @@ def meshgrid(*xi, copy=True, sparse=False, indexing='xy'):
     >>> yv
     array([[0.,  0.,  0.],
            [1.,  1.,  1.]])
-    >>> xv, yv = np.meshgrid(x, y, sparse=True)  # make sparse output arrays
+
+    The result of `meshgrid` is a coordinate grid:
+
+    >>> import matplotlib.pyplot as plt
+    >>> plt.plot(xv, yv, marker='o', color='k', linestyle='none')
+    >>> plt.show()
+
+    You can create sparse output arrays to save memory and computation time.
```

Comment:
Are there any times / reasons where using `sparse` is problematic / not worth it? I'd assume there's a size at which it becomes more useful, perhaps something to be mentioned?

Reply:
Hmmm I'm not sure about this. I'll try to investigate.